com.unity.render-pipelines.high-definition/Documentation~/Master-Node-Unlit.md

```diff
@@ -27,10 +27,10 @@ The following table describes the input ports on an Unlit Master node, including
 | ----------------------------------------- | -------- | --------- | ------------------------------------------------------------ |
 | **Position**                              | Vector 3 | Vertex    | Defines the absolute object space position of the Material per vertex.|
 | **Color**             		            | Vector 3 | Fragment  | Defines the color of the Material. To assign an image, connect a Sample Texture2D to this Master node. |
-| **Alpha**				                    | Vector 1 | Fragment  | Defines the Material's alpha value. Used for transparency and/or alpha clip. Expected range 0 - 1.|
-| **AlphaClipThreshold**                    | Vector 1 | Fragment  | Set the alpha value limit that HDRP uses to determine whether it should render each pixel of the Material. If the alpha value of the pixel is equal to or higher than this threshold then HDRP renders the pixel. If the value is lower than this threshold then HDRP does not render the pixel. This port only appears when you enable the **Alpha Clipping** setting.|
+| **Alpha**				                    | Float    | Fragment  | Defines the Material's alpha value. Used for transparency and/or alpha clip. Expected range 0 - 1.|
+| **AlphaClipThreshold**                    | Float    | Fragment  | Set the alpha value limit that HDRP uses to determine whether it should render each pixel of the Material. If the alpha value of the pixel is equal to or higher than this threshold then HDRP renders the pixel. If the value is lower than this threshold then HDRP does not render the pixel. This port only appears when you enable the **Alpha Clipping** setting.|
 | **Emission**                           	| Vector 3 | Fragment  | Defines the Material's emission color value. Expects positive values.|
-| **Shadow Tint**                              | Vector 4 | Fragment  | Defines the shadow color value with alpha value. Enable only if Shadow Matte enable cf. [Master node settings menu](#SettingsMenu)|
+| **Shadow Tint**                           | Vector 4 | Fragment  | Defines the shadow color value with alpha value. Enable only if Shadow Matte enable cf. [Master node settings menu](#SettingsMenu)|
```

Comment:
I'm not quite sure what this means: "Enable only if Shadow Matte enable cf."

Reply:
this is not related to the change in this PR but it mean that Shadow tint property is only having an effect if shadow matte property is enabled.

Reply:
I rephrased it as follows:
> Defines the shadow color value with alpha value. The **Shadow Tint** property only has an effect if you enable the **Shadow Matte** property located in the [Master node settings menu](#SettingsMenu).